### ISSUE TITLE
Resolve theme accessibility issues to meet WCAG 2.0 AA

### ISSUE THREAD

**Issue by priscillamc** [ufclas/ufclas-ufl-2015#17]
Text Alternatives

- Images with empty alt text in the featured post carousel. 
- Add alternative text to image links

Adaptable

- Properly tag the slides in the featured post carousel (See [role=listbox](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_listbox_role))
- Distinguish non-unique IDs. #query, #source, #site, #site-footer
- Improve non-distinguishable landmarks. search, complementary (See [aria-labelledby](https://www.w3.org/TR/WCAG20-TECHS/ARIA13))
- Include all content in landmarks ([Using ARIA landmarks](https://www.w3.org/TR/WCAG20-TECHS/ARIA11))

Distinguishable

- Color contrast is not sufficient (white text on orange background)

Enough Time

- Make sure that the user can influence any time limits by stopping it, adjusting it or extending it.

Navigable

- Image link has no alternative text
- Read More link text is not sufficient, not distinguishable
- Keyboard focus must be visible

Input Assistance

- Select box without a description
- Field label is not visible

Compatible

- Make sure that WAI-ARIA role matches functionality (button, listbox, etc)
